addon/errors.ts

```diff
@@ -1,6 +1,4 @@
-import EmberError from '@ember/error';
-
-export class AjaxError extends EmberError {
+export class AjaxError extends Error {
```

Comment:
to be a bit more verbose here: Ember 3.8 changed `EmberError` to no longer be an `Error` subclass, but instead reexport `Error`. Babel 6 can't deal with extending native classes, but Babel 7 can. Babel 7 does not detect that it needs to do that though because of the reexport.

Since the bump to `ember-cli-babel@7` and the corresponding change in the minimum Ember CLI requirements require a major version bump anyway it seems fine to change the base class here to a regular `Error` too.

Reply:
you just solved why we couldn't figure out subclassing with that change properly in ember-data (not detecting the native class due to re-export)

Reply:
happy to help 😉